### ISSUE TITLE
dateMath.parse is not a function

### ISSUE THREAD

**Issue by sunfeilong1993** [grafana/kairosdb-datasource#10]
use "grafana3.0.0-beta2" and "kairosdb-datasource" on windows
When i click "This Week" “This Month” and "This Year", I get an error “dateMath.parse is not a function”

```

boot.js:36 TypeError: dateMath.parse is not a function
    at convertToKairosTime (datasource.js?bust=1461123434286:426)
    at KairosDBDatasource.performTimeSeriesQuery (datasource.js?bust=1461123434286:65)
    at KairosDBDatasource.query (datasource.js?bust=1461123434286:55)
    at h.f.issueQueries (boot.js:31)
    at h.issueQueries (boot.js:17)
    at h (boot.js:36)
    at boot.js:36
    at n.$eval (boot.js:36)
    at n.$digest (boot.js:36)
    at n.$apply (boot.js:36)

```

when i use "grafana-2.6.0",  this error is disappeared

**Comment by Weijianlin** [comment]
use kairosdb-datasource and pie chart

TypeError: dateMath.parse is not a function
at convertToKairosTime (datasource.js?bust=1460967128164:426)
at KairosDBDatasource.performTimeSeriesQuery (datasource.js?bust=1460967128164:65)
at KairosDBDatasource.query (datasource.js?bust=1460967128164:55)
at PieChartCtrl.f.issueQueries (boot.8698c185.js:31)
at h (boot.8698c185.js:36)
at boot.8698c185.js:36
at n.$eval (boot.8698c185.js:36)
at n.$digest (boot.8698c185.js:36)
at boot.8698c185.js:36
at e (boot.8698c185.js:35)

**Comment by strotter** [comment]
Also seeing this same issue with Grafana 3.0.3 and latest Kairos plugin.

**Comment by strotter** [comment]
@sunfeilong1993 I have fixed this issue and made a pull request: https://github.com/grafana/kairosdb-datasource/pull/13

**Comment by sunfeilong1993** [comment]
@strotter  Thank you, this problem is disappeared .

**Comment by strotter** [comment]
@sunfeilong1993 We are now having more severe issues with this plugin in Grafana 3.0.3 after fixing this bug, building a new dashboard and graph panel creates invalid queries that aren't downsampled/aggregated properly and are showing datapoints that don't make sense... have you run into anything similar?  And what version of Kairos are you on with Grafana 3.0?

**Comment by sunfeilong1993** [comment]
Sorry,  this bug  is not disappeared. Because I did not choose any data source so when I create a new graph panel this bug is not appear. But when I choice Kairos data source tthis bug appeared again. Maybe I was not using the latest version.

"building a new dashboard and graph panel creates invalid queries that aren't downsampled/aggregated properly and are showing datapoints that don't make sense." If you mean is “Create a valid query, but get some data”, I did not encounter this problem。

`
Windows 10
grafana-3.0.0-beta2
kairosdb-1.1.1-1 with cassandra-3.4
`

My English is poor, I am sorry, If you do not know what I mean.
